app/controllers/step_by_step_pages_controller.rb

```diff
@@ -77,8 +78,10 @@ def publish
   def schedule
     set_current_page_as_step_by_step
     if request.post?
-      date_params = params[:schedule][:date].permit(:year, :month, :day)
-      parser = DatetimeParser.new(date: date_params, time: params[:schedule][:time])
+      date_params = params[:schedule][:date].permit(:year, :month, :day).to_h.symbolize_keys
+      time_param = params[:schedule][:time]
+      @schedule_placeholder = default_datetime_placeholder(date_params.merge(time: time_param))
```

Comment:
Can `params[:schedule]` be used here? e.g. `default_datetime_placeholder(params[:schedule])`